src/LinkMe/LinkMe.Svelte/app.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+console.log('Hello world');
```

Comment:
we may need `app.js` and `app.js.map` to be excluded from GIT, since they are generated by TS tool.

not sure if all JavaScript files should be excluded, I don't have enough knowledge about LinkMe languages.